pandas/core/dtypes/dtypes.py

```diff
@@ -326,6 +326,9 @@ def _validate_categories(categories, fastpath=False):
             if not categories.is_unique:
                 raise ValueError('Categorical categories must be unique')
 
+        if isinstance(categories, ABCCategoricalIndex):
```

Comment:
here you an also accept ABCCategorical

Reply:
a Categorical has already been converted to CategoricalIndex

Reply:
ahh ok